zone/gm_commands/guild.cpp

```diff
@@ -476,9 +502,215 @@ void command_guild(Client *c, const Seperator *sep)
 			}
 		}
 	}
+	else if (is_details)
```

Comment:
@Kinglykrab have suggestions for this section ?

Reply:
I would personally use a popup for the data type stuff.

Seems there's a lot of unnecessary reformatting here as well.

Reply:
Updated the #guild info and the DescribeGuild function.